packages/react-components/react-drawer/stories/Drawer/DrawerBestPractices.md

```diff
@@ -14,6 +14,8 @@
 ### Accessibility
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31570&commitId=a5719bc6056da66cd5f0bd6855f82b465f143bb7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|1| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|5| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Can't see any difference